manifests/pipeline.yaml

```diff
@@ -1,53 +1,67 @@
 apiVersion: v1
 kind: Template
 labels:
-  app: fedora-coreos
-  template: fedora-coreos-template
+  app: ${DISTRO}-coreos
+  template: ${DISTRO}-coreos-template
 metadata:
   annotations:
     description: |-
-      Jenkins pipeline for Fedora CoreOS.
+      Jenkins pipeline for CoreOS distributions.
     iconClass: icon-jenkins
     openshift.io/display-name: Fedora CoreOS Pipeline
     openshift.io/documentation-url: https://github.com/coreos/fedora-coreos-pipeline
     openshift.io/support-url: https://github.com/coreos/fedora-coreos-pipeline
-    openshift.io/provider-display-name: Fedora CoreOS
-    tags: fcos,jenkins,fedora
-  name: fedora-coreos
+    openshift.io/provider-display-name: fedora CoreOS
+    tags: ${TAGS}
+  name: ${DISTRO}-coreos
 parameters:
   - description: Git source URI for Jenkinsfile
     name: REPO_URL
     value: https://github.com/coreos/fedora-coreos-pipeline
   - description: Git branch/tag reference for Jenkinsfile
     name: REPO_REF
     value: master
+  - description: Distribution name
+    name: DISTRO
+    value: fedora
+  - description: Tags
+    name: tags
```

Comment:
is there a reason to leave this parameter lowercase, vs uppercase like the others?

Reply:
Nope, fixed and repushed. I also caught that the default value needed to be corrected.